tests/contrib/test_rest_framework_dataclasses.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import typing
+
+from dataclasses import dataclass
+from django.urls import path
+from drf_spectacular.utils import extend_schema
+from rest_framework.decorators import api_view
+from rest_framework_dataclasses.serializers import DataclassSerializer
+
+from tests import assert_schema, generate_schema
+
+
+@dataclass
```

Comment:
probably a good idea to move the dataclasses and the dataserializer inside the test (including the imports). we need to support 3.6 till April just as Django will.